docs/BUILD_PIPILINE_DEBUGGING.md

```diff
@@ -0,0 +1,91 @@
+# Build Pipeline Debugging
+
+**Note**: This is _only_ for debugging purposes and should not be relied upon.
+
+To gain more visibility into what broccoli trees Ember CLI operates on, you can run the following command:
+
+```sh
+BROCCOLI_DEBUG="bundler:*" ember build
+```
+
+Fish equivalent is:
+
+```sh
+env BROCCOLI_DEBUG="bundler:*" ember build
+```
+
+For a brand new ember application (`ember new your-app-name`), `DEBUG/` folder is going to be created with the following contents:
+
+```sh
+DEBUG/
+└── bundler:application-and-dependencies
+    ├── addon-tree-output
```

Comment:
Not sure that we need to list all the contents here, perhaps just a `tree -L 2 DEBUG/` (2 levels) or something?